build.gradle

```diff
@@ -22,7 +22,7 @@ import groovy.time.TimeCategory
 buildscript {
     repositories {
         mavenCentral()
-        mavenLocal()
+//        mavenLocal()
```

Comment:
@alexander-aleksandrov FYI. I have fixed that already in this branch (see my recent commits).